src/RubyGosu.cxx

```diff
@@ -8057,6 +8095,37 @@ _wrap_Song_current_song(int argc, VALUE *argv, VALUE self) {
 }
 
 
+SWIGINTERN VALUE
+_wrap_Song_length(int argc, VALUE *argv, VALUE self) {
```

Comment:
Wait...what? I must have messed around with all the audio branches too badly because Audio.hpp declares length() methods that are never implemented in Audio.cpp. Can you please remove them from the header files, and re-run SWIG?

Reply:
Done and done.